src/shared/components/molecules/BulkActionProgress.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import styled from '../../lib/styled'
+import ProgressBar from '../../../components/atoms/ProgressBar'
+import { keyframes } from 'styled-components'
+import { BulkActionState } from '../../../cloud/components/molecules/ContentManager/ContentManagerToolbar'
+
+interface BulkActionProgressProps {
+  bulkActionState: BulkActionState | null
+}
+
+function getProgressPercentage(progress: BulkActionState) {
+  return Math.max(
+    Math.min((progress.jobsCompleted / progress.jobCount) * 100, 100),
+    0
+  )
+}
+
+const BulkActionProgress = ({ bulkActionState }: BulkActionProgressProps) => (
+  <Container>
+    <ProgressBarContainer>
+      <ProgressItems>
+        <Spinner />
```

Comment:
Custom spinner with white color since the previous spinner which I copied is not customizable - I would like to make it so.

The other cloud spinner is more versatile, but needs positioning, do you have any suggestions, should I leave it like this or should we use the already available cloud spinner and position it?

Reply:
We should use the existing spinner rather than having another spinner.

Reply:
Okay, I'll try to position the old spinner, hopefully it will work nicely